packages/block-editor/src/components/rich-text/index.js

```diff
@@ -336,6 +336,7 @@ class RichTextWrapper extends Component {
 			didAutomaticChange,
 			undo,
 			placeholder,
+			keepPlaceholderOnFocus,
```

Comment:
Should we include the deprecation message?

Reply:
We are not actually deprecating this one.

Reply:
We need to restore documentation then.

Reply:
#17445